hnn_core/network_builder.py

```diff
@@ -552,12 +568,19 @@ def get_data_from_neuron(self):
             isoma_py[gid] = {key: rec_i.to_python()
                              for key, rec_i in rec_i.items()}
 
+        lfp_py = list()
+        for elec in self._lfp:
+            # elec.pc.allreduce(elec.lfp_v, 1)
+            # lfp_py.append(elec_list[pos].lfp_t.to_python())
```

Comment:
Maybe you meant to remove these? Also, realize that you may have been testing out `allreduce` where my comment above about efficiency. Please ignore if that's the case. Main point is just that commented lines could be removed.